src/core/core.scale.js

```diff
@@ -8,6 +8,7 @@ import {autoSkip} from './core.scale.autoskip.js';
 
 const reverseAlign = (align) => align === 'left' ? 'right' : align === 'right' ? 'left' : align;
 const offsetFromEdge = (scale, edge, offset) => edge === 'top' || edge === 'left' ? scale[edge] + offset : scale[edge] - offset;
+const getTicksLimit = (ticksLength, maxTicksLimit) => Math.min(Math.max(maxTicksLimit || ticksLength, 0), ticksLength);
```

Comment:
I would assume ticksLegnth is never null and either 0 or bigger. never negative so the inner math.max would be redundent

Reply:
@LeeLenaleee not sure if not needed. The inner `Math.max` is apply for for `maxTicksLimit`, not for `ticksLength`.  
Maybe I'm wrong (please correct me) but if `maxTicksLimit` (options set by user) is negative (and could be), without the inner `Math.max`, the result will be a negative value which is not correct.

Reply:
True it could be negative but that is user error in my eyes and an unsupported way of working.
I dont know why you would want to set it to a negative number, would you expect for ticks to be doubled in that case?

Reply:
> would you expect for ticks to be doubled in that case?

No no... that's my mindest.. if I can avoid an user error, I prefer doing it. Let me update it, removing the inner max.

Reply:
Its not wrong, but it adds in my eyes unecesarry complexity. and without it, it reads a lot easyer which I prefer

Reply:
removed! Thank you @LeeLenaleee